enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,381 @@
+the two (VPC and Power VS). This is a high-speed link meant to connect IBM DCs and other colocated datacenters.
+Each of these things (VPC, networks, and Direct Link connection) will be created via the IBM Cloud Terraform provider.
+
+#### DNS
+The Power VS provider will use the same DNS solution as the IBM Cloud provider: DNS provided by
+[IBM CIS][ibm-cis-website]. A user will be required to create a DNS domain in CIS before beginning an install and
+provide it to the installer.
+
+Further information about CIS can be found in the [official docs][ibm-cis-docs-website].
+
+### Persistent Storage
+At launch, the planned support for storage (for PV & PVCs) is NFS & ODF as day 2 configuration options.
+On the roadmap we would looking into [Spectrum Scale][ibm-spectrum-scale-website] as a day 2 configuration as well as a
+CSI driver for Power VS. These features would look to be enabled in a future release of OpenShift.
+
+### CoreOS OVA
+When we started, there were two main implementation ideas for making the CoreOS OVA boot image available to the Virtual
+Server instance. At this point we've settled on design 1, below, because it can be fully automated and betters meets
+the expectations of an IPI experience.
+
+1. Import the Image from a Cloud Store This is the implementation we believe is best, since it works most similarly to
+other cloud environments. In this setup, the CoreOS build pipeline would publish an OVA image to an IBM Cloud Object
+Storage (COS) bucket. Then, we will use the regular IBM Cloud APIs to import the image to the instance. The drawback to
+this approach is the the current implementation is very slow. In its current state, it takes about 30 minutes to import
+an 120 Gb boot image. This is too long to meet our target of being able to install a whole cloud within the hour. One of
+the reasons this is slow is because we cannot do the image resizing after the copy, since it's writing directly to a
+sized disk. Once imported, creating the nodes is fast, since the other disks are flashed from the snapshot. For now, we
+asked the team to look into meeting the requirement that importing the image can take 5 minutes or less. Another bit
+we're working on is defining the implementation details for how the image will be imported from the COS bucket to the
+PowerVS service. Currently, the terraform API doesn't support importing an image from bucket. We are looking into if it
+is possible to update the terraform provider to do this with support for optional keys to support both private and
+public buckets. IBM Cloud Object Storage supports public buckets, but the terraform provider API will need an update to
+make keys optional in order to support this workflow.
+
+This will happen as part of the terraform provider.
+
+~~2. Published CoreOS Images to the IBM Cloud Pre-Authorized Default Images This approach would involve outputting our
```

Comment:
We are actually doing something similar to this on other platforms at the moment, I think this may be the preferred approach in the long term

Reply:
agree this is preferred, but as we don't use the same process for images as ibmcloud, and the power vs folks who handle the image publication process don't have a mechanism for outsiders to provide publicly-available images, this may have to be very long-term. we can open an enhancement request with them (if we haven't already. i need to double-check)

Reply:
now are we talking about "boot" images here or "disk" images ( the OVAs)..i believe when we talk about pre-authorized default images, it is the actual bootimage and not the OVAs. this would be a good distinction to make in the enhancement that what is provided by the RHCOS pipeline today and consumed by the installer is the disk image and not the bootimage.

Reply:
I've reworked this section since this isn't under consideration because we don't have a viable way to automate it.

Perhaps we can follow this up with additional enhancements if we can get the Power VS cloud team to add mechanisms for this.